src/librbd/librbd.cc

```diff
@@ -586,6 +586,11 @@ namespace librbd {
     librbd::image_options_create_ref(&opts, opts_);
   }
 
+  ImageOptions::ImageOptions(ImageOptions &imgopts)
```

Comment:
Thanx @trociny , Actually I agree with you here. But I have to wait #12102 to be merged. Then I will rebase this branch and address the comments all. Thanx for your kindly help.